bundles/org.openhab.binding.wiz/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,208 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="wiz" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="color-bulb">
+		<label>WiZ Color Bulb</label>
+		<description>Supports WiZ Full Color with Tunable White Bulbs</description>
+
+		<channels>
+			<channel id="color" typeId="system.color"/>
+			<channel id="temperature" typeId="system.color-temperature"/>
+			<channel id="temperature-abs" typeId="system.color-temperature-abs"/>
+			<channel id="mode" typeId="light-mode"/>
+			<channel id="speed" typeId="light-mode-speed"/>
+			<channel id="signal-strength" typeId="system.signal-strength"/>
```

Comment:
Do you think it's worth adding the RSSI as an advanced channel, since it's available and the signal strength is actually derived from that?

Reply:
Yes, I think that's a great idea.